pkgs/by-name/de/deltachat-tauri/package.nix

```
@@ -0,0 +1,108 @@
+{
+  apple-sdk_14,
+  cargo-tauri,
+  darwin,
+  fetchFromGitHub,
+  gst_all_1,
+  lib,
+  libayatana-appindicator,
+  makeWrapper,
+  nodejs,
+  openssl,
+  perl,
+  pkg-config,
+  pnpm_9,
+  python3,
+  rustPlatform,
+  stdenv,
+  versionCheckHook,
+  webkitgtk_4_1,
+  wrapGAppsHook4,
+}:
+
+let
+  pnpm = pnpm_9;
+in
+rustPlatform.buildRustPackage rec {
+  pname = "deltachat-tauri";
+  version = "1.59.1";
+
+  src = fetchFromGitHub {
+    owner = "deltachat";
+    repo = "deltachat-desktop";
+    tag = "v${version}";
+    hash = "sha256-wHpEny8gOBYuNWKGO7NzfH7u7CO8l4bnJkNPYfcL88Q=";
+  };
+
+  pnpmDeps = pnpm.fetchDeps {
+    inherit pname version src;
+    hash = "sha256-WUk6PoTSHDL/9wZVwDLksfuIBs8hTxjG4aeAMDUaB8o=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-PntpKkF5Z7lXd3zIbuIdI2m9iNmQkFqidTKuOUJT6zs=";
```

can be updated to 1.60.1

```suggestion
  version = "1.60.1";

  src = fetchFromGitHub {
    owner = "deltachat";
    repo = "deltachat-desktop";
    tag = "v${version}";
    hash = "sha256-WrP4C4zebbRdxy08oQzR3vkO2IBUfsRPnkU1aWqolB4=";
  };

  pnpmDeps = pnpm.fetchDeps {
    inherit pname version src;
    hash = "sha256-PBCmyNmlH88y5s7+8WHcei8SP3Q0lIAAnAQn9uaFxLc=";
  };

  useFetchCargoVendor = true;
  cargoHash = "sha256-3pU8hxUNaP3hlauA1D3yJiwektcILtGcinFgQDaA0L0=";
```

since 1.60.1 the menu bar items texts (i.e. File, Edit etc.) are offset with a left margin of several pixels and i don't know if it's supposed to have icons there or something